cli/tools/registry/graph.rs

```diff
@@ -103,7 +103,7 @@ pub fn surface_fast_check_type_graph_errors(
             );
             break 'analyze_package; // no need to keep analyzing this package
           } else {
-            let message = diagnostic.message_with_range();
+            let message = diagnostic.message_with_range_for_test();
```

Comment:
Looks like this method got removed, but I assume @lucacasonato you're diagnostic rewrite is gonna change it anyway?

Reply:
Yes just land this and I'll address it in the diagnostics PR